### PR TITLE
Remove CSS formatting from spotless apply

### PR DESCRIPTION
**Pull request by DanVanAtta**
Remove 'postcss' formatter, it keeps running into 501 errors when
spotlessApply is run locally:

```
Execution failed for task ':spotlessStyling'.
> Unexpected response status code at /prettier/format [HTTP 501] -- (Error while formatting: Error: Couldn't resolve parser "postcss")
```

## Functional Changes

[] New map or map update
[] New Feature
[] Feature update or enhancement
[] Feature Removal
[] Code Cleanup or refactor
[x] Configuration Change
[] Problem fix
[] Other:

### CODE REVIEW

**Comment by DanVanAtta** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- game-core/src/main/java/games/strategy/net/nio/Decoder.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5909536&bid=19359121)

File: build.gradle [hunk 1]
```diff
@@ -45,10 +45,6 @@ allprojects {
             indentWithSpaces()
             trimTrailingWhitespace()
         }
-        format 'styling', {
-            target '**/*.css'
-            prettier().config(['parser': 'postcss'])
```
Comment (RoiEXLab) [inline build.gradle:50]: The parser is now just called `css`, in case you want to change it

Comment (DanVanAtta) [reply build.gradle:50]: No joy simply changing it:

```
Execution failed for task ':spotlessStyling'.
> Unexpected response status code at /prettier/format [HTTP 501] -- (Error while formatting: TypeError: text.trimEnd is not a function)
```

Comment (RoiEXLab) [reply build.gradle:50]: Ah, it's this error again. It seems like the newer 'prettier' requires a newer version of NodeJS, which introduces a `String.trimEnd` function.
But nevermind then

**Pull request merged**